R/calculateAverageWidths.R

```diff
@@ -0,0 +1,126 @@
+get_pavement_widths = function(road_geojson_path, roadside_geojson_path, crs = "EPSG:27700", buffer_dist = 15, segment_length = 20) {
```

Comment:
This function is not ready to merge. Just adding here to highlight it, we should migrate it in a separate PR right @wangzhao0217 ?